website/404.html

```diff
@@ -0,0 +1,18 @@
+
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="styles.css"/>
+    <title>Crow Fact Oracle&trade;</title>
```

Comment:
```suggestion
    <title>Crow Fact Oracle</title>
```
Not sure what the `&trade;` is?

Reply:
It's the trademark symbol! :D
![tm](https://user-images.githubusercontent.com/29528527/78540134-8783e300-77a8-11ea-867d-7b3e35ab3a84.png)